bot/main.py

```diff
@@ -1,15 +1,20 @@
+import argparse
 import asyncio
 import logging
 import sys
-from .constants import AppCommands
-import argparse
-from .messages import make_help_message, make_daily_messages
+
+from dotenv import load_dotenv
```

Comment:
No need to use dotenv. There's already `pydantic-settings` that loads the environment. I expect the code to know nothing about `.env` files.